functions.sh

```diff
@@ -109,12 +109,15 @@ function create_image_version_from_disk() {
 		--os-type Linux \
 		--os-state generalized
 
+    image_timestamp=$(date --date=@$timestamp_in_seconds +'%Y.%m.%d%H%M%S')
+    timestamp=$(date --date=@$timestamp_in_seconds +'%Y%m%d%H%M%S')
```

Comment:
These lines are not sufficiently indented. Sorry I missed this during my previous review.